### PR TITLE
Reintegrate Redux into Gatsby

### PR DESCRIPTION
**Pull request by ironng**
Makes the forms work again!

- Moves redux initialization into a new `AppWrapper` component, used by both gatsby-browser.js and gatsby-ssr.js'
- Re-connects the Home component to the redux store
- Cleans up discord/email signup code (removes dedicated action for discord signup and creates a handler in the view for it, etc)
- Updates indentation for touched files 

(Images, nav, and some layout are still broken, they will be fixed in separate PRs)

### CODE REVIEW

**Comment by ironng** [comment]
Works locally in both development mode and running the production build...but not working in the preview 🤔:weary:

**Comment by ironng** [comment]
@Shadowfiend This is ready for review. The issue was the missing `pathPrefix` property for the gatsby-config. Requests to the backend are going to fail on the preview site b/c need to whitelist cross origin requests from preview.keep.network in the backend repo.

**Comment by Shadowfiend** [comment]
Hmmm… I'm wondering if we want to drag redux back in here, or if it's better to just use the Fetch API directly… But in the interest of keeping things relatively close to what they were I guess we can stick with what's here.

File: src/components/AppWrapper.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+export default ({ element }) => {
+  // Instantiating store in `wrapRootElement` handler ensures:
+  //  - there is fresh store for each SSR page
+  //  - it will be called only once in browser, when React mounts
+  const sagaMiddleware = createSagaMiddleware()
+  const store = createStore(sagaMiddleware)
+
+  sagaMiddleware.run(saga)
+  store.dispatch(actions.createSession())
```
Comment (Shadowfiend) [inline src/components/AppWrapper.js:15]: hello darkness, my old friend

Comment (ironng) [reply src/components/AppWrapper.js:15]: 😂

**Comment by Shadowfiend** [comment]
Actually I don't want to block on the backend question above. If you're ready to merge I'll hit the button @ironng !

**Comment by ironng** [comment]
Let's merge! We can add that handler to the backend soon!

**Pull request merged**